tests/lib/devicetree/devices/testcase.yaml

```diff
@@ -15,6 +15,7 @@ tests:
       - bl5340_dvk_cpuapp
       - bl5340_dvk_cpuapp_ns
       - m5stack_core2
+      - m5stack_atoms3lite
```

Comment:
might be worth mentioning why in the commit message?

Reply:
Actually, when reviewing I saw that this is not really needed. Removed it from the devicetree test in [bd019a9](https://github.com/zephyrproject-rtos/zephyr/commit/bd019a9cf65db130247357c3d9de3a8da0d7a504)